libp2p/NodeTable.h

```diff
@@ -127,8 +127,13 @@ class NodeTable: UDPSocketEvents, public std::enable_shared_from_this<NodeTable>
 	using NodeSocket = UDPSocket<NodeTable, 1280>;
 	using TimePoint = std::chrono::steady_clock::time_point;	///< Steady time point.
 	using NodeIdTimePoint = std::pair<NodeID, TimePoint>;
-	using EvictionTimeout = std::pair<NodeIdTimePoint, NodeID>;	///< First NodeID (NodeIdTimePoint) may be evicted and replaced with second NodeID.
-	
+	struct EvictionTimeout
+	{
+		NodeID m_leastSeenID;
```

Comment:
Please remove `m_` prefix. The fields are public. Also place `NodeID` field next to each other.